assets/scss/_custom.scss

```diff
@@ -242,9 +242,23 @@ body.td-404 main .error-details {
 .search-item.nav-item {
     input, input::placeholder {
       color: black;
+      width: 200px;
+      box-sizing: border-box;
+      padding: 8px;
+      height: 40px;
+      font-size: 14px;
+      transition: width 0.3s ease-in-out;
```

Comment:
Why a `transition`?

Reply:
> Why a `transition`?

I have used the transition property to facilitate the smooth transition when the width of the search box changes, and it leads to a better user experience. 

Reply:
Ok, cool.